psychopy/visual/window.py

```diff
@@ -3105,36 +3105,53 @@ def getActualFrameRate(self, nIdentical=10, nMaxFrames=100,
 
         """
         if nIdentical > nMaxFrames:
-            raise ValueError('nIdentical must be equal to or '
-                             'less than nMaxFrames')
+            raise ValueError(
+                'Parameter `nIdentical` must be equal to or less than '
+                '`nMaxFrames`')
+
+        # log that we're measuring the frame rate now
+        msg = "{}: Attempting to measure frame rate of screen ({:d}) ..."
+        logging.exp(msg.format(self.name, self.screen))
+
+        # Only log changing `recordFrameIntervals` if differs from the user's
+        # settings. Disable recording for the warmup period.
         recordFrmIntsOrig = self.recordFrameIntervals
-        # run warm-ups
-        self.recordFrameIntervals = False
+        self.setRecordFrameIntervals(False, log=logging.DEBUG)
```

Comment:
The `log` arg is a Bool indicating whether or not to override the autolog` setting for this Window, so setting it to `logging.DEBUG` has no effect

Reply:
I supposed we could use it to change the level of logging, as you've tried here, but there would be quite a lot of lines to refactor then